src/Module/Tos.php

```diff
@@ -16,6 +16,27 @@
 
 class Tos extends BaseModule
 {
+	// Some text elements we need more then once to keep updating them easy.
+	public $privoperate;
```

Comment:
For clarity, can you please expand/rename those variable names to:
- `privacy_operate`
- `privacy_distribute`
- `privacy_delete`
- `privacy_complete`